packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -119,6 +119,7 @@
 			// ... and display labels.
 			&::after {
 				content: attr(aria-label);
+				font-size: 0.75rem;
```

Comment:
Can you use `$helptext-font-size` here instead of rem units? We use this same variable (which returns 12px) for form widget help text.

Reply:
Sure! I didn't find that variable (it's not with the other font sizes at the top of the file), so thought I'd sneakily introduce a responsive unit instead 😄